CONTRIBUTING.md

```diff
@@ -337,6 +337,84 @@ We are using ES-Lint to maintain code quality & consistency in our UI Code. To m
 - We will soon be adding Pre-Commit Hooks to make sure you get to know your errors before you commit the code.
 - In case you are unable to fix your lint errors, ping us on our [Slack](https://slack.meshery.io).
 
+
+# Using Sistent in Meshery UI
+
+## Overview
+
+Meshery UI utilizes three component libraries:
+
+1. Material-UI (MUI) v4
+2. Material-UI (MUI) v5
+3. Sistent
+
+While MUI v4 and v5 are being phased out, Sistent is now the preferred component library. Sistent internally uses MUI v5, and Meshery UI globally still relies on MUI v5. This can lead to conflicts between themes when Sistent components are used directly.
```

Comment:
```suggestion
While MUI v4 and v5 are being phased out, Sistent is now the preferred component library. Sistent internally uses MUI v5, and Meshery UI globally still relies on MUI v4. This can lead to conflicts between themes when Sistent components are used directly.
```